### ISSUE TITLE
3.9 instantly closing

### ISSUE THREAD

**Issue by kyslmao69** [DIGITALCRIMINAL/OnlyFans#199]
Running StartDatascraper.py opens then instantly closes. Re-downloaded a couple times, still no luck.
Running latest version of pip and python. 3.8 still works tho.

**Comment by DIGITALCRIMINAL** [comment]
Have you edited your config and install all requirements?

**Comment by NotMyMainOne** [comment]
I'm having the same problem. Only edits to config were the usual ones and all requirements are installed.

```
Z:\completed\Other\onlyfans>py startdatascraper.py

Z:\completed\Other\onlyfans>
```

Yeah I don't know why but since reinstalling my system I need to type py instead of python to start a script. This has been working for 3.8 though.

**Comment by DIGITALCRIMINAL** [comment]
Are you both using Linux?

**Comment by NotMyMainOne** [comment]
Yes for me. Well my host computer is Windows 10 with python and everything installed but the Onlyfans folder with everything is on my Unraid machine being accessed via network drive.

**Comment by DIGITALCRIMINAL** [comment]
> Yes for me. Well my host computer is Windows 10 with python installed but the Onlyfans folder with everything is on my Unraid machine being accessed via network drive.

Could you remove line one of this code? This might be the reason. It was added a couple of weeks ago.
![image](https://user-images.githubusercontent.com/21285176/79050116-bb835c00-7c1f-11ea-8b12-4b57393216ad.png)

**Comment by NotMyMainOne** [comment]
That made it start but authorization fails and after the 20 attempts says "You're not subscribed to any users." before starting over again. Double checked and all information in the config is correct.

**Comment by DIGITALCRIMINAL** [comment]
Have you got the latest commit? I pushed one 3 hours ago which changed the auth and added new things to the config

**Comment by NotMyMainOne** [comment]
Just tried it and same error but showing differently. Instead of 

```
Auth (V1) Attempt 1/10
Access Denied
Auth (V1) Attempt 2/10
Access Denied
```

It's showing:

```
Auth (V1) Attempt 1/10
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Access Denied
Auth (V2) Attempt 1/10

```

Do any new settings need to be adjusted in config first this time?

**Comment by DIGITALCRIMINAL** [comment]
The auth is, all the same, I added two things to the config that doesn't affect the auth process.
Do you have 2FA on your account?

**Comment by NotMyMainOne** [comment]
I do not.

**Comment by DIGITALCRIMINAL** [comment]
I'm getting that error now. I'll fix it.

**Comment by DIGITALCRIMINAL** [comment]
I fixed it by updating my user-agent....

**Comment by kyslmao69** [comment]
For me, just using windows on a pc

**Comment by NotMyMainOne** [comment]
Thanks mate. I had tried moving the folder to my Desktop and just change the directory in the config but got the same error so I figured it was some weird change.

**Comment by DIGITALCRIMINAL** [comment]
@NotMyMainOne oh so it's working now?
@kyslmao69 are you using the latest commit?

**Comment by NotMyMainOne** [comment]
Sorry no I was just saying I was trying it there too but just tried updating the user-agent on both and neither worked. :(

**Comment by DIGITALCRIMINAL** [comment]
Literally the only thing I can think of is 2FA .-.

**Comment by kyslmao69** [comment]
@DIGITALCRIMINAL Just downloaded latest commit, still autocloses. 8.8 still works fine.

**Comment by DIGITALCRIMINAL** [comment]
Are you calling the script from CMD or just clicking the .py file @kyslmao69

**Comment by DIGITALCRIMINAL** [comment]
@kyslmao69 run StartDatascraper.py in the Python 3.8 IDLE

**Comment by NotMyMainOne** [comment]
Just want to update that after rolling back to 3.8 tonight I got the same error, then I checked OF again and realized the auth_hash had changed. I input the new auth_hash and it worked so I went and tried it on 3.9 and it worked there as well. So I'm thinking there was some funky change of the auth_hash that didn't show up in my browser earlier in the day but did later on. Trust me I checked it over 100x when we were troubleshooting and the auth_hash hadn't changed in the network tab.

Also I want to note 3.9 did in fact fix the issue we had in 3.8 with that error due to the text format or whatever it was on some accounts. So everything is working perfect in 3.9 latest commit. Thanks for the help and maintaining this whole thing. It's a great service.
